library/dokku_network_property.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# from ansible.module_utils.basic import *
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  network:
+    description:
+      - The name of the network to attach to
+    required: True
+    default: null
+    aliases: []
+  property:
+    description:
+      - The attach network property
+    required: True
+    default: null
+    choices: [ "initial-network", "attach-post-create", "attach-post-deploy" ]
+    aliases: []
+author: Philipp Sessler
+requirements: [ ]
+"""
+
+EXAMPLES = """
+- name: Associates a network after a container is created but before it is started
+  dokku_network_property:
+    app: hello-world
+    network: example-network
+    property: attach-post-create
+
+- name: Associates the network at container creation
+  dokku_network_property:
+    app: hello-world
+    network: example-network
+    property: initial-network
+
+- name: Setting a global network property
+  dokku_network_property:
+    app: --global
+    network: example-network
+    property: attach-post-create
+
+- name: De-associate the container with the network.
+  dokku_network_property:
+    app: hello-world
+    network: example-network
+"""
+
+
+def dokku_network_property_set(data):
+    is_error = True
+    has_changed = False
+    meta = {"present": False}
+
+    command = "dokku network:set {0} {1} {2}".format(
+        data["app"],
+        data["property"] if "property" in data else "",
```

Comment:
Is this the right logic?

I'm reading [here](https://dokku.com/docs/networking/network/#attaching-an-app-to-a-network) that one should leave the value empty in order to de-associate the container from the network rather than leave the name of the property empty.

Reply:
Yes, you're completely right. I mixed that up